supply-chain-security/compromises/2019/monero.md

```diff
@@ -0,0 +1,21 @@
+<!-- cspell:ignore monero -->
+# Binaries of the CLI for `monero` compromised
+
+The binaries of the CLI wallet had been compromised and a malicious version
+was being served. The attacker introduced two new functions. Anyone who
+created or opened a wallet with the compromised binary had their seed stolen.
+
+## Impact
+
+The impact of this incident is unknown however at least one Reddit user claimed
+to have lost funds worth $7000.
+
+## Type of Compromise
+
+This incident fits the [Trust and Signing](../compromise-definitions.md#trust-and-signing) definition.
```

Comment:
I think this may be more appropriate for [publishing infrastructure](https://github.com/cncf/tag-security/blob/main/supply-chain-security/compromises/compromise-definitions.md#publishing-infrastructure). Other than that, LGTM!

Reply:
Ah, yes that make sense